Specs/Core/EllipsoidRhumbLineSpec.js

```diff
@@ -0,0 +1,581 @@
+defineSuite([
+        'Core/EllipsoidRhumbLine',
+        'Core/Cartographic',
+        'Core/Ellipsoid',
+        'Core/EllipsoidGeodesic',
+        'Core/Math'
+    ], function(
+        EllipsoidRhumbLine,
+        Cartographic,
+        Ellipsoid,
+        EllipsoidGeodesic,
+        CesiumMath) {
+    'use strict';
+
+    it('throws without start', function() {
+        expect(function() {
+            var rhumb = new EllipsoidRhumbLine();
+            return rhumb.interpolateUsingSurfaceDistance(0);
+        }).toThrowDeveloperError();
+    });
+
+    it('throws without end', function() {
+        expect(function() {
+            var rhumb = new EllipsoidRhumbLine(new Cartographic(Math.PI, Math.PI));
+            return rhumb.interpolateUsingSurfaceDistance(0);
+        }).toThrowDeveloperError();
+    });
+
+    it('throws without unique position', function() {
+        expect(function() {
+            var rhumb = new EllipsoidRhumbLine(new Cartographic(Math.PI, Math.PI), new Cartographic(0, Math.PI));
+            return rhumb.interpolateUsingSurfaceDistance(0);
+        }).toThrowDeveloperError();
+    });
+
+    it('setEndPoints throws without start', function() {
+        expect(function() {
+            var rhumb = new EllipsoidRhumbLine();
+            rhumb.setEndPoints();
+        }).toThrowDeveloperError();
+    });
+
+    it('setEndPoints throws without end', function() {
+        expect(function() {
+            var start = new Cartographic(CesiumMath.PI_OVER_TWO, 0);
+            var rhumb = new EllipsoidRhumbLine();
+            rhumb.setEndPoints(start);
+            return rhumb.interpolateUsingSurfaceDistance(0);
+        }).toThrowDeveloperError();
+    });
+
+    it('can create using fromStartAndEnd function', function() {
+        var fifteenDegrees = Math.PI / 12;
```

Comment:
Not a huge deal, but might be worth declaring these frequently-used angles in one place up at the top of the spec file.